wordpress-mobile/WordPress-Android
# Issue/4888 notifications allow horizontal nav always

Fixes #4888 

(Note the functionality to limit horizontal navigation was there on purpose, so we got rid of it now after having users expect things to work differently)

To test:
1. being logged in to the app, have some other user make a comment on a blog post of a blog you're an admin of
2. you should receive a notification for such comment
3. tap on the comment, the detail view will be shown.
4. you should be able to swipe right to see more notifications in your list (and then left/right accordingly as you navigate along)

cc @nbradbury
